clients/src/test/java/org/apache/kafka/test/TestSslUtils.java

```
@@ -56,6 +56,24 @@
 
 public class TestSslUtils {
 
+    /*
+     * Simulates a CA signing certificates
+     */
+    static KeyPair issuerKP;
+    static AsymmetricKeyParameter privateKeyAsymKeyParam;
+    static X509Certificate issuerCert;
+    static X500Name issuer = new X500Name("CN=localhost, O=CAroot");
+    static {
+        try{
+            issuerKP = generateKeyPair("RSA");
+            privateKeyAsymKeyParam = PrivateKeyFactory.createKey(issuerKP.getPrivate().getEncoded());
+            issuerCert = generateCertificate("CN=localhost, O=CAroot", issuerKP, 30,
+                    "SHA1withRSA", true);
+        } catch (Exception e) {
+
```

Need to deal with this empty catch block.